### PR TITLE
Accept JSON as application/json or text/plain

### PR DESCRIPTION
**Pull request by switters**
So we can avoid CORS preflight and hold IE9's delicate hand

Wrap the JSON.parse in a try...catch and do something like this to the accept middleware:

```js
function accept () {
  var types = ['text/plain', 'application/json']
  return function (req, res, next) {
    var type = req.headers['content-type']
    var err = (~types.indexOf(type)) ? null : {message:'not ok', status:415}
    res.set('Accept', types)
    next(err)
  }
}
```

### CODE REVIEW

File: lib/http.js [hunk 1]
```diff
@@ -41,6 +43,9 @@ function cors (req, res, next) {
  */
 
 function rawBody(req, res, next) {
```
Comment (lvivier) [inline lib/http.js]: are you planning to remove this middleware? why use body-parser and this one, they do the same thing?

**Comment by switters** [comment]
@lvivier :rocket:

**Comment by switters** [comment]
@lvivier :rocket: :man: ?

**Comment by lvivier** [comment]
:guitar: :+1:

**Pull request merged**